### ISSUE TITLE
Could not install platform

### ISSUE THREAD

**Issue by etcwebdesigns** [platformio/platformio-home#1257]
PIO Core Call Error: "Platform Manager: Installing c:/user/babyg/downloads/Seeed_Arduino_LIS3DHTR-master.zip\r\n\n\nError: VCS: Unknown repository type c:/user/babyg/downloads/Seeed_Arduino_LIS3DHTR-master.zip"

**Comment by etcwebdesigns** [comment]
I am not sure that I am doing this correctly: I have a Seeed-Arduino 3 Axis accelerometer and the library is the LIS3DHTR but this library is not in the PlatformIO library. So I tried to install it, I wasn't sure how to install it until I ran across this section of custom install. I don't know if this is just for the IDE or not but it seemed to make sense. I even tried to install it using the Git web page for the repository but that didn't work either. Any help would be awesome.
